modules/nft/src/lib.rs

```diff
@@ -65,7 +65,7 @@ pub struct ClassData<Balance> {
 	pub attributes: Attributes,
 }
 
-#[derive(Encode, Decode, Clone, RuntimeDebug, PartialEq, Eq, TypeInfo, Serialize, Deserialize)]
+#[derive(Encode, Decode, Clone, RuntimeDebug, PartialEq, Eq, TypeInfo, Serialize, Deserialize, Default)]
```

Comment:
is the Default necessary? I want it to be explicit

Reply:
`Default` removed